app/helpers/sessions_helper.rb

```diff
@@ -1,18 +1,22 @@
 module SessionsHelper
-  # Logs in the goven user.
   def log_in(user)
     session[:user_id] = user.id
   end
 
-  # Return the current logged-in user (if any).
-
   def logged_in?
     !current_user.nil?
   end
 
-  # Logs out the current user.
   def log_out
     session.delete(:user_id)
     @current_user = nil
   end
+
+  def logged_in_user
+    unless logged_in?
```

Comment:
Is this method used anywhere? Can't find a call. Also message can be changed if it works already :p